dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/RumWebChromeClient.kt

```diff
@@ -20,6 +20,7 @@ import com.datadog.android.rum.RumErrorSource
  * Any console message will be forwarded to an internal [Logger], and errors
  * will be sent to the [GlobalRum] monitor as RUM Errors.
  */
+@Deprecated("You should use the DatadogEventBridge JavaScript interface instead.")
```

Comment:
Maybe we should add a snippet or indication here on _how to use_ the `DatadogEventBridge`.

Reply:
I was thinking to add this in a separated task once we merge this in master. What do you think ?

Reply:
Well let's do it now, that we we won't forget it